tutorials-and-examples/inference-servers/jetstream/maxtext/single-host-inference/README.md

```diff
@@ -93,22 +93,22 @@ kubectl create secret generic kaggle-secret \
 
 ## Convert the Gemma-7b checkpoint
 
-To convert the Gemma-7b checkpoint, we have created a job `checkpoint-job.yaml` that does the following:
+To convert the Gemma-7b checkpoint, we have created a job that does the following:
 1. Download the base orbax checkpoint from kaggle
 2. Upload the checkpoint to a Cloud Storage bucket
 3. Convert the checkpoint to a MaxText compatible checkpoint
 4. Unscan the checkpoint to be used for inference
 
 In the manifest, ensure the value of the BUCKET_NAME environment variable is the name of the Cloud Storage bucket you created above. Do not include the `gs://` prefix.
 
-Apply the manifest:
+Apply the manifest from the root of this repo to create the job:
 ```
-kubectl apply -f checkpoint-job.yaml
+BUCKET_NAME=<YOUR_BUCKET_NAME> IMPLEMENTATION="JAX" ./modules/checkpoint-converter/bash-equivalent/apply.sh
```

Comment:
If you want to use bash scripts, you should migrate them to this jetstream maxtext single-host-inference directory.

A modules directory is a terraform standard naming schema indicating that the files within it are terraform scripts.

In this case, these bash scripts are specific tooling that works mostly for this tutorial rather than what is indicated to be reusable terraform modules.